README.md

```diff
@@ -1,2 +1,202 @@
-# saritasa-github-actions
-GitHub actions
+## CI-wordpress reusable workflow
+
+### Deploying
+
+In order to deploy, you need to perform some steps:
+
+- Create a new playbook for deploying your app, for example, `deploy.yml`
+- Place the created playbook in your repository for example, `ci\ansible\deploy.yaml`
+- Set up variables for workflow (see Inputs Variables)
```

Comment:
Input Variables - should be either link to the workflow or to another section in this README.md file

Reply:
done